usecase/lunchbreak/lunch.py

```diff
@@ -18,17 +19,40 @@ class Lunchbreak:
     start = None
     end = None
 
+    calender_remote = None
+
     def __init__(self, mock:bool=False):
         if mock:
+            # self.set_WeatherApapter(WeatherMock())
```

Comment:
Are these comments necessary ?

Reply:
it is to be discussed if this is the future implementation

Reply:
I guess that's something we need to discuss with all @mariushegele @fionnfuchs @JannisTriesToCode what do you guys think?